core/src/main/scala/chisel3/ElaboratedCircuit.scala

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import firrtl.annotations.Annotation
+import firrtl.ir.{CircuitWithAnnos, Serializer}
+import firrtl.options.{CustomFileEmission, Unserializable}
+import chisel3.experimental.{BaseModule, SourceInfo, UnlocatableSourceInfo}
+import chisel3.experimental.hierarchy.Definition
+import chisel3.internal.firrtl.ir.Circuit
+import chisel3.internal.firrtl.Converter
+
+/** The result of running Chisel elaboration
+  *
+  * Provides limited APIs for inspection of the resulting circuit.
+  */
+// This is an important interface so let's keep it separate from the implementation.
```

Comment:
Does this comment cause problems with Scaladoc?

Reply:
It does not!